latex/ledger-spec.tex

```diff
@@ -1483,7 +1481,7 @@ \subsection{Witnesses}
       \forall i \in \txins{tx} \cdot \exists! (\var{vk}, \sigma) \in \txwits{\var{tx}}
       \cdot \\
      \mathcal{V}_{\var{vk}}{\serialised{\txdata{tx}}}_{\sigma}
-     \wedge  \fun{addr_h}~{utxoSt}~i = \hashKey{vk}\\
+     \wedge  \fun{addr_h}~{utxo}~{pointeraddr}~i = \hashKey{vk}\\
```

Comment:
I think this no longer works for base addresses. I think we need a function that takes either a pointer address or a base address, and returns the payment hashKey. this is a bit awkward for writing total functions since reward addresses are addresses as well, and they do not have payment keys.